### DISCUSSION TITLE
Adding support for CRC32?

### DISCUSSION THREAD

**Discussion by InterClaw** [WinMerge/winmerge#1992] [Ideas]
Would you consider adding support for CRC32?

**Comment by sdottaka** [comment]
WinMerge does not support CRC32, but it is possible to add columns such as MD5 to the folder comparison window as follows.
![image](https://github.com/WinMerge/winmerge/assets/98126/3f24ecd4-08a6-4b0d-a2b6-166c20cbb0b0)

**Comment by InterClaw** [comment]
Yes, that's exactly what I found as well and felt that there was something missing from that picture... ;)
